java-errorreporting/versions.txt

```diff
@@ -2,5 +2,5 @@
 # module:released-version:current-version
 
 google-cloud-errorreporting:0.124.4-beta:0.124.5-beta-SNAPSHOT
```

Comment:
Are you handling the `-beta-SNAPSHOT` versions correctly?

Reply:
Ah, thanks for catching that. I'll update the regex.